docs/source/gbd2017_models/causes/covid_19/index.rst

```diff
@@ -11,35 +11,285 @@ Covid-19 Custom SEIR Vivarium Cause Model
 ++++++++++++++++++++++++++++++++
 
 State Data
 ^^^^^^^^^^
 
+Prevalence
+""""""""""
+
+Simulants will be initialized into covid-19 disease model states based on the forecasted IHME covid-19 model estimates. 
+
+.. todo::
+
+	Detail specific data source and values
+
+Mortality
+"""""""""
+
+We will model covid-19 mortality using an age-dependent infection fatality ratio (IFR), as consistent with the IHME forecast model. There will be two steps to this piece of the model, the first being the determination of if an individual dies due to covid-19 (described here), and the being determining *when* they exit their current model state (decribed in the `Transition Data`_ section).
+
+In the same time-step for which a simulant is initialized into the :math:`E`, :math:`I_1`, or :math:`I_2` covid-19 model states or transitions into the :math:`E` model state, the age-specific IFR should be used to determine if that simulant will die of covid-19, such that the IFR shown in the table below represents the probability that an infected individual dies from covid-19. 
+
+.. csv-table:: Infection Fatality Ratios
+	:file: ifr_data.csv
+	:header-rows: 1
+
+.. note::
+
+	This data was obtained from an email communication with Abie from May 5, 2020. These values may need to be updated as more recent numbers become available.  
+
 Transition Data
 ^^^^^^^^^^^^^^^
 
+Susceptible to Exposed
+""""""""""""""""""""""
+
+Method One
+~~~~~~~~~~
+
+This method of modeling the susceptible to exposed state transition in the covid-19 model will be a replication of the transition probability used in the IHME forecast model such that the **incidence rate for an individual simulant at time-step t is defined as:**
+
+.. math:: \frac{\beta(t)S(I_1+I_2)^\alpha}{N}
+
+Where,
+
+.. list-table:: Parameter Definitions
+   :widths: 5 25
+   :header-rows: 1
+
+   * - Parameter
+     - Definition
+   * - :math:`S`
+     - Number of simulants in the :math:`S` state at time-step :math:`t`
+   * - :math:`I_1`
+     - Number of simulants in the :math:`I_1` state at time-step :math:`t`
+   * - :math:`I_2`
+     - Number of simulants in the :math:`I_2` state at time-step :math:`t`
+   * - :math:`N`
+     - Total number of simulants in the model at time-step :math:`t`
+   * - :math:`\beta(t)`
+     - Taken from IHME forecast model under stated assumptions regarding covariates
+   * - :math:`\alpha`
+     - Taken from IHME forecast model under stated assumptions
+
+.. note::
+
+	This method assumes random mixing of the population and assumes homogeneous transmission probability across all demographic groups.
+
+Method Two
+~~~~~~~~~~
+
+This method of modeling the susceptible to exposed state transition in the covid-19 model uses the :math:`\beta(t)` parameter *scaled to the average duration of the infectious period* OR :math:`R_t` from the IHME forecast model under stated assumptions regarding coviariates as the simulation's :math:`R_0` value under those same stated assumptions.
```

Comment:
When you say 'OR', does that mean that those two things are actually equivalent, or that there is a 'Method 2a' and a 'Method 2b' embedded here? They sound like they both aim to be the true R_t, but might be two methods of estimating R_t/might yield different values (?)